core/src/main/java/org/apache/accumulo/core/conf/cluster/ClusterConfigParser.java

```diff
@@ -83,8 +83,7 @@ private static void flatten(String parentKey, String key, Object value,
   public static void outputShellVariables(Map<String,String> config, PrintStream out) {
     for (String section : SECTIONS) {
       if (config.containsKey(section)) {
-        out.println(
-            String.format(PROPERTY_FORMAT, section.toUpperCase() + "_HOSTS", config.get(section)));
+        out.printf((PROPERTY_FORMAT) + "%n", section.toUpperCase() + "_HOSTS", config.get(section));
```

Comment:
```suggestion
        out.printf(PROPERTY_FORMAT + "%n", section.toUpperCase() + "_HOSTS", config.get(section));
```

Reply:
Not sure why, but I missed this and following comments - will address in next PR.